File: 2018/day1/day1.swift
import Foundation

var data = try NSString(contentsOfFile: "2018/day1/day1input.txt",
    encoding: String.Encoding.ascii.rawValue)
let inputs = data.components(separatedBy: .newlines)
var total = 0
for i in inputs {
    print(i)
    let number = Int(i)
    total = total + number!
}
print(total)

File: 2018/day1/day1.py
import io
total = 0

with io.open("day1input.txt", 'r', encoding='utf-8') as f:
    for line in f:
        operator = line[0]
        number = int(line[1:].rstrip())
        if operator == "+":
            total += number
        if operator == "-":
            total -= number
print(total)

